File: code-coaches/easy/popsicles/Popsicles.cs
using System;

public class Popsicles
{
    static string DistributePopsicles(int popsicles, int siblings)
    {
        if (popsicles % siblings == 0)
			Console.WriteLine("give away");
		else
			"eat them yourself");
    }

	static void Main(string[] args)
	{
		int siblings;

		do
		{
			siblings = Convert.ToInt32(Console.ReadLine());
		}
		while (siblings == 0);

		int popsicles = Convert.ToInt32(Console.ReadLine());

		Console.WriteLine(DistributePopsicles(popsicles, siblings));
	}

}

File: code-coaches/easy/popsicles/Popsicles.swift
func distributePopsicles(popsicles: Int, siblings: Int) -> String {
    if popsicles % siblings == 0 {
		return "give away"
	} else {
		return "eat them yourself"
	}
}

func main() {
	let siblings = Int(readLine()!)!
	let popsicles = Int(readLine()!)!
	print(distributePopsicles(siblings: siblings, popsicles: popsicles))
}

main()